pyo3-macros-backend/src/method.rs

```diff
@@ -218,6 +218,7 @@ impl CallingConvention {
 }
 
 pub struct FnSpec<'a> {
+    pub asyncness: bool,
```

Comment:
I think it would improve readability if we put this next to `unsafety` and handle it similarly w.r.t. the type.